engine/admin/container_priority.md

```diff
@@ -0,0 +1,117 @@
+---
+description: Limit a container's access to CPU resources or running the container as a real-time process
+keywords:
+- priority, real time, nice, resources, cpu
+title: Configure a container's CPU scheduling
+---
+
+Containers run as isolated processes on the Docker daemon host. Normally a
+container gets access to CPU resources proportional to all other running
+processes which need CPU resources. If your host machine and Docker daemon are
+configured correctly, you can schedule Linux container tasks to have higher
+priority access to CPU resources. If you need to, you can even schedule
```

Comment:
The options Docker provides are really more about being able to configure the resources that the RT scheduler should allot for it, but does not set the scheduler itself.